### ISSUE TITLE
version 3.4 assertion comparison wrong

### ISSUE THREAD

**Issue by JackKuo-tw** [rocky/python-uncompyle6#289]
## Description

assertion comparison wrong in version 3.4.0
but it's correct in version 3.3.0

## How to Reproduce

source:
```python3
c = 3
assert c < 2, ('Oops')
if c < 2:
    raise SyntaxError('Oops')
```

decompile from version 3.4.0
```python3
c = 3
assert c < 2, 'Oops'
assert c < 2, 'Oops'
```

## Expected behavior

decompile from version 3.3.0
```python3
c = 3
if not c < 2:
    raise AssertionError('Oops')
if c < 2:
    raise SyntaxError('Oops')
```

so version 3.4 should be:
```python
c = 3
assert c < 2, 'Oops'
assert not(c < 2), 'Oops'
```

## Environment

- Uncompyle6 version: 3.4.0
- Python version: 3.6.8
- OS and Version: Ubuntu bionic

**Comment by rocky** [comment]
Thanks for the report. Think I have it fixed in 9bd85fe . In the future I might not have the time to address, would you be interested in learning how to fix problems like this?

**Comment by JackKuo-tw** [comment]
What a fast fix!!!
Thanks for your invitation, I have no time to do this project too :'(
One reason is my knowledge about python byte code is insufficient.
Hope you can find the volunteers~
